ynput/OpenPype
# Fusion: Output Image Format Updating Instances (create_saver)

## Changelog Description
Adds the ability to update Saver image output format if changed in the Publish UI.

~~Adds an optional validator that compares "Output Image Format" in the Publish menu against the one currently found on the saver. It then offers a repair action to update the output extension on the saver.~~

## Additional info
The current behaviour allows users to change the "Output Image Format" in the Publish menu without reflecting these changes in the comp/scene. In my view wherever changes are possible users will expect those to have an impact. This change remedies that.

~~This optional validator addresses this by informing the user when a mismatch occurs between the Publish UI and the saver. It also offers a repair action to update the output format/extension to that of the Publish UI. This is a draft just to gage if this is a good fit or if an alternative approach to solving this is better.~~

## Testing notes:
1. Open Fusion.
2. Create new 'Render' variant.
3. Open Publish menu.
4. Edit the "Output Image Format" to a different value.
5. Publish.
6. Verify that the new path on saver is correct and has updated to new value previously selected in step 4.
